### ISSUE TITLE
Why I use the same device (two 2080ti GPUs) can only set the batchsize to 2？

### ISSUE THREAD

**Issue by mdswyz** [WangLiwen1994/DeepRelight#4]
Hello！ First of all, thank you for your open source code, your project is great. But when I was reproducing your code, I could only set the batchsize to 2. I noticed that you stated in the paper that your mini-batch size is 6, use two 2080ti, I also use the same device to run but the batchsize can only be set to 2 (this is the default in the code), so are there any other settings that need to be changed?

**Comment by WangLiwen1994** [comment]
Yes, we also faced the memory problem last year. Our solution is that: [1] We trained the Scene Reconversion and Shadow Prior Estimation Networks individually. [2] we loaded the parameters back (see ''models/networks.py" line 225-228) and removed the last output layers. (see ''models/networks.py" line 304 and 390). [3] we disabled the gradient of the two networks to save GPU memory and finetuned the whole network (see ''models/networks.py", line 219-223, you may need to uncomment the lines). Optionally, you may use half-precision as another choice.

**Comment by mdswyz** [comment]
Thank you very much for your prompt reply, I will try immediately！
